OpenRA.Mods.Common/Traits/PowerTooltip.cs

```diff
@@ -0,0 +1,50 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Shown power info on the build palette widget.")]
+	public class PowerTooltipInfo : UpgradableTraitInfo
```

Comment:
This inherits from `UpgradableTraitInfo`, but `PowerTooltip` does not inherit from `UpgradableTrait`. The simple solution is to replace `UpgradableTraitInfo` with `ITraitInfo` and remove `override` from `Create`.